### ISSUE TITLE
Need for 'Test_definitions' folder under 'code'

### ISSUE THREAD

**Issue by Kevsy** [camaraproject/WorkingGroups#302]
Problem description:
As discussed in #61  , there is a minimum requirement for  a `.feature` file to satisfy the test criteria for API Readiness.

The discussion [includes the point](https://github.com/camaraproject/WorkingGroups/issues/61#issuecomment-1584472138) "Everyone seem to agree that .feature file in main subproject will help with compliance of minimum checklist criteria document."

It seems sensible to place thie `.feature` file(s) in a folder under `code`, i.e. `/code/Test_definitions'

**Comment by Kevsy** [comment]
Thanks, too many repos open! Will delete this and re-open in Commonalities.

**Comment by rartych** [comment]
@Kevsy Yes, we should update the subproject template repository, not only with tests placeholder, but also with Github issue templates etc.

The issue in Commonalities subproject is the right way,  then my colleagues from DT can update https://github.com/camaraproject/Template_Lead_Repository - hopefully before opening new subprojects.

In case of existing repositories it should be manually keeping the folder structure decided in Commonalities.
